csharp/ql/src/Stubs/Stubs.qll

```diff
@@ -513,12 +513,30 @@ private string stubClassName(Type t) {
                                   stubClassName(element), "," order by i
                                 ) + ")"
                         else
-                          if t instanceof ValueOrRefType
+                          if t instanceof FunctionPointerType
```

Comment:
You can remove this if, and instead replace the matching `else` with `or not t instanceof FunctionPointerType and`

Reply:
Sure - we can do it like that instead - I just continued the pattern that was already there.
However, it might make things slightly more complicated, when adding new cases 😄